SECURITY.WHITEPAPER.md

```diff
@@ -0,0 +1,129 @@
+# Datasafe Security Whitepaper
+
+## General information
+Datasafe is a flexible encryption library. It uses different encryption algorithms. Some of them can be 
+configured by client application. Under the hood Datasafe uses BouncyCastle library to perform encryption.
+
+CMS (Cryptographic Message Syntax) standard [RFC5652](https://tools.ietf.org/html/rfc5652.html) employed for storing private 
+files encrypted with symmetric keys as well as for sharing files with other users using asymmetric key pairs. 
+
+## Default locations
+By default on system start have to be configured one system dfs (distributed file system).
```

Comment:
Hard to understand, maybe:
To maintain information where user data resides, Datasafe uses concept of user profile. There are two types of user profile: private and publc...

Also this is not really relevant for security, it is better to hide it and show as collapsible note